firmware/tunerstudio/rusefi.input

```diff
@@ -1220,6 +1222,15 @@ menuDialog = main
 		subMenu = oilPressureSensor,		"Oil pressure"
 		subMenu = egtInputs,				"EGT" @@if_ts_show_egt
 
+menu = "&E-Throttle"
+
+        subMenu = etbDialog,				"E-Throttle Settings" @@if_ts_show_etb
```

Comment:
Use the full word - "Electronic Throttle Body"

Reply:
I can compromise with Electronic Throttle ?